.circleci/config.yml

```diff
@@ -88,6 +88,12 @@ aliases:
         git checkout -B "$CIRCLE_BRANCH" "$CIRCLE_SHA1"
       fi
 
+      # Piggyback on this alias to enable VNC connections
```

Comment:
Hmm. I can see this being a bit confusing. I wouldn't expect this checkout alias to also setup VNC connections. And there are some jobs where we don't use this alias because we need more git history.

Could we move this to a separate alias?

Reply:
Resolved by renaming the alias instead